R/scrape_authors.R

```diff
@@ -77,8 +77,15 @@ vt_scrape_tags_from <- function(type, tags = c("editor","editDate"), src = ".",
           }, block)
 
         item <- block$object$topic
+        if(is.null(item)){
 
-        do.call('data.frame', setNames(c(item, tag_values, FALSE), c(type, tags,"stringsAsFactors")))
+          abort(message = paste0("File ",
+                                  block$object$file,
+                                  " must include @title if deprecated."),
+                class = "vt.test_code_format")
```

Comment:
Could we add a line/which test in the file? I think that would be more informative if possible.